content/docs/setup/kubernetes/prepare/platform-setup/openshift/index.md

```diff
@@ -39,3 +39,31 @@ constraints as part of sidecar injection:
 {{< text bash >}}
 $ oc adm policy add-scc-to-user privileged -z default -n <target-namespace>
 {{< /text >}}
+    
+## Automatic Injection
+Webhook and certificate signing requests support must be enabled for auto injection to work. To do so, you must modify the master configuration file for the cluster. 
```

Comment:
auto->automatic.  Please link to the automatic injection page.  Remove "To do so".

Reply:
Thanks. Updated.